setup.py

```diff
@@ -24,10 +24,11 @@
             'pyosf', 'requests[security]',
             'xlrd', 'openpyxl',
             'pyyaml', 'gevent', 'msgpack-python', 'psutil', 'tables'
+            'opencv-python',
             ]
 # some optional dependencies
 if platform == 'darwin':
-    required.append('pyobjc-core')
+    required.append('pyobjc-core', 'pyobjc-framework-Quartz')
```

Comment:
Sorry, I'm a newbie at reviews. Do you mean submit a review and request changes? Or do you want me to submit a PR to your branch?